Tests/SwiftFusionTests/Core/VectorTests.swift

```diff
@@ -322,4 +222,28 @@ class MultipleVectorTests: XCTestCase {
     XCTAssertEqual(pb(Vector5(0, 0, 0, 1, 0)).1, Vector3(0, 1, 0))
     XCTAssertEqual(pb(Vector5(0, 0, 0, 0, 1)).1, Vector3(0, 0, 1))
   }
+
+  func testConvertToTensor() {
```

Comment:
Is this really part of vector semantics testing, or could you just not find a more suitable place to put it?

Reply:
It is testing some methods that are implemented as extensions of `Vector`. Maybe changing the class name to `VectorExtensionTests` clarifies this?